src/entities/questionnaire.ts

```diff
@@ -184,8 +184,19 @@ export function getFilteredQuestions(
     }
 
     const filteredQuestions = questions.filter((question) => {
+        const jointMoreTitles = question.moreTitles
+            ? Object.values(question.moreTitles).join(' \n ') : '';
```

Comment:
Why have space before and after `\n`?
Did `\n` work as expected?

Reply:
because it might return us with results for 'n'+ some other string

Reply:
Yeah it works as expected